pages/common/gh-gist.md

```diff
@@ -3,22 +3,22 @@
 > Work with GitHub Gists on the command-line.
 > More information: <https://cli.github.com/manual/gh_gist>.
 
-- Create a new Gist from a space-separated list of files:
+- Create a new Gist from specific files:
 
-`gh gist create {{path/to/files}}`
+`gh gist create {{path/to/file1 path/to/file2 ...(1..)}}`
 
-- Create a new Gist with a description:
+- Create a new Gist with a specific [desc]ription:
 
-`gh gist create {{path/to/file}} --desc "{{description}}"`
+`gh gist create {{path/to/file1 path/to/file2 ...(1..)}} --desc "{{description}}"`
 
-- Edit a Gist:
+- Edit a specific Gist:
 
-`gh gist edit {{id_or_url}}`
+`gh gist edit {{id|url}}`
```

Comment:
This might imply that the literal text `id` or `url` is desired here, which is not the case.

Let's change this back to the old syntax.

```suggestion
`gh gist edit {{id_or_url}}`
```

Reply:
Old change may imply that the literal `id_or_url` is expected.